<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>660</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>221</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="x0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X0</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="x0_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="y0">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>68</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y0</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="y0_input">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>68</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Xg</string>
    </property>
   </widget>
   <widget class="QLabel" name="yg">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>68</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Yg</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="xg_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="yg_input">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>添加直线轨迹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_arc">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>添加顺圆弧轨迹</string>
    </property>
   </widget>
   <widget class="QSlider" name="speed">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="r_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>68</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>慢</string>
    </property>
   </widget>
   <widget class="QLabel" name="r_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>68</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>快</string>
    </property>
   </widget>
   <widget class="QLabel" name="r_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>直线或者圆弧的起点</string>
    </property>
   </widget>
   <widget class="QLabel" name="r_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>直线或者圆弧的终点</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="origin_x">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="origin_y">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="yg_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>圆心坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_arc_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>添加逆圆弧轨迹</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuX0">
    <property name="title">
     <string>comp</string>
    </property>
   </widget>
   <addaction name="menuX0"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
